p2panda-core/src/lib.rs

```diff
@@ -1,5 +1,70 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 
+//! Core data types used across the p2panda stack to offer distributed, secure and efficient data
+//! transfer between peers.
+//!
+//! The main data type is a highly extensible, cryptographically secure append-only log
+//! implementation. It provides all the basic features required to implement more advanced
+//! distributed data types commonly required when building peer-to-peer and local-first
+//! applications.
+//!
+//! # Features:
+//!
+//! - cryptographic signatures for authorship verification and tamper-proof messages
+//! - various ordering algorithms can be applied over collections of messages
+//! - provides mechanisms for efficient sync of past state
+//! - is compatible with any networking scenario (even broadcast-only)
+//!
+//! p2panda logs are made up of [`Operation`]s. Authors sign operations using their cryptographic
+//! keypair and append them to a log. An author may have one or many logs. The precise means of
+//! identifying logs is not defined by this crate (see extensions).
+//!
+//! A common challenge in distributed systems is how to order operations written concurrently by
+//! different authors and/or processes. Operations contain information which can be used for
+//! establishing order depending on one's use case:
+//! - `timestamp`: The UNIX timestamp describing when the operation was created.
+//! - `previous`: An (optional) list of hashes referring to the previously observed operations can
+//!   be used to establish cryptographically secure partial-ordering.
+//!
+//! It is worth noting that ordering algorithms are _not_ further specified or provided as part of
+//! `p2panda-core`.
```

Comment:
As a reader I've stumbled upon this sentence. I believe we _specified_ ordering algorithms, but indeed, they are not implemented. On top, I'd wonder now where I would need to look if they're not in `p2panda-core`?

Since this sentence is maybe just confusing, should we remove it? Or say that we'll have specifications for ordering algorithms and will provide implementations soon (maybe as part of `p2panda-core`, I think we didn't decide that yet)?